sdks/go/pkg/beam/core/timers/timers.go

```diff
@@ -53,6 +53,7 @@ type TimerMap struct {
 
 type timerConfig struct {
 	Tag           string
+	HoldSet       bool
```

Comment:
In this case, it's relating to whether the "Hold Timestamp was Set By the User", so the Set is more about "Get+Set" rather than the mathematical or computer science notion of Set. I can add a comment (will resolve when changed).

Reply:
I would have chosen `isHoldSet`. Now it makes sense why below in the earlier code `!tc.HoldTimestamp.ToTime().IsZero()` is the behavior equivalent of `HoldSet` or `isHoldSet`.